bridges/primitives/messages/src/lib.rs

```diff
@@ -102,6 +102,18 @@ impl TypeId for LaneId {
 	const TYPE_ID: [u8; 4] = *b"blan";
 }
 
+impl From<LaneId> for u32 {
```

Comment:
@NachoPal @svyatonik 
I am not sure about this, can we move this out of `bridges` substree?
Because then we need to backport it back to bridges repo and also in new version LaneId has a different type: https://github.com/paritytech/parity-bridges-common/pull/2221/files#diff-836b204328629e6af4b33c2a34ae41191cab06cf89c066e31bdb7085314fcda1L185-R203

Reply:
Agreed there's probably a better way to do this.

Reply:
The problem is implementing `From` for `LaneId` or the fact that `LaneId` is going to change? I do not think is harmful to implement `From` for a new type.

I could try to locally (in `xcm-emulator`) wrap `LaneId` in a new type and implement the `From` trait for it.

Reply:
> I could try to locally (as part of the BridgeHubMessageHandler impl) wrap LaneId in a new type and implement the From trait for it.

@bkontur @gilescope Check it out